modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/UniqueCompositeKey.tsx

```diff
@@ -376,68 +392,60 @@ export function UniqueCompositeKey({
 					filter={true}
 					firstColumnHeader={Liferay.Language.get('label')}
 					onDeleteColumn={(objectFieldName) => {
-						const makeFetch = async () => {
-							const objectValidation: ObjectValidation = await API.getObjectValidationRuleById(
-								values.id as number
-							);
-
-							const canNotDeleteObjectField = builderScreenItems.some(
-								(builderScreenItem) =>
-									(objectValidation.objectValidationRuleSettings as ObjectValidationRuleSetting[]).some(
-										(objectValidationRuleSetting) =>
-											objectValidationRuleSetting.value ===
-											builderScreenItem.externalReferenceCode
-									) &&
-									builderScreenItem.objectFieldName ===
-										objectFieldName &&
-									(objectDefinition as ObjectDefinition)
-										.status.label === 'approved'
-							);
+						const canNotDeleteObjectField = builderScreenItems.some(
```

Comment:
@igor-franca I don't think we need to uppercase the "Not" here. It can be `cannotDeleteObjectField`.